extensions/amp-ad-network-cloudflare-impl/0.1/cloudflare-a4a-config.js

```diff
@@ -23,7 +23,9 @@
  */
 export function cloudflareIsA4AEnabled(win, element) {
   // check for an explicit attribute to enable A4A processing.
-  // This could be just true, if all ads from this network are A4A, or any
-  // other logic based upon configuration, client settings, etc.
-  return !!element.getAttribute('data-a4a') && element.hasAttribute('src');
+  // This defaults to true, but allows disabling A4A for ads known
```

Comment:
Done.  Also, this was already in _a4a-config.js from a previous PR.  We're just generifying the implementation as a courtesy to our customers.